flow-server/src/main/resources/vite.generated.ts

```diff
@@ -180,7 +180,9 @@ function vaadinBundlesPlugin(): PluginOption {
 
   const disabledMessage = 'Vaadin component dependency bundles are disabled.';
 
-  const modulesDirectory = path.posix.resolve(__dirname, 'node_modules');
+  const modulesDirectory = path.resolve(__dirname, 'node_modules');
+  // always refine path to comply with URI path separator (forward slash) for requested resources:
+  const modulesDirPathSepRefined = path.sep === '/' ? modulesDirectory : modulesDirectory.replace(/\\/g, '/');
```

Comment:
This is a bit extra as it could just be 
```suggestion
  const modulesDirectory = path.resolve(__dirname, 'node_modules').replace(/\\/g, '/');
```